DEP Tasks/Trainers Task/JSCore/Task 1/Transportation on vacation.js

```diff
@@ -0,0 +1,12 @@
+function rentalCarCost(d) {
+    // Your solution here
+    if(d>=7){
+      return amount=(d*40)-50;
```

Comment:
Where did `amount` variable has been declared?

Should use single return statement in a function where ever possible. So, remove multiple return statements.

Reply:
done